src/streamlink/plugins/kanal7.py

```diff
@@ -0,0 +1,33 @@
+"""
+$description Turkish live TV channel.
+$url kanal7.com
+$type live
+"""
+
+import logging
+import re
+
+from streamlink.plugin import Plugin, pluginmatcher
+from streamlink.plugin.api import validate
+from streamlink.stream.hls import HLSStream
+
+log = logging.getLogger(__name__)
+
+
+@pluginmatcher(re.compile(r"""
+    https?://(?:www\.)?
+    (?:
+        kanal7\.com/canli-izle
+    )
+""", re.VERBOSE))
+class KANAL7(Plugin):
+
```

Comment:
Why the uppercase class name? The blank line below is also unnecessary and might result in a linting error in the future.

Reply:
I based my plugin on the tv8.plugin. The class name was written in capital letters there.